3drepo/3drepo.io
# Issue 4636 - Toolbar: transform reset

This fixes #4636 

#### Description
Added the logic to clear transformations (not just from sequences)

#### Test cases
- Select a view, open an issue/risk/ticket with default view, or select the date of a sequence that has transformations: The "reset transformations" buttons should show up in the toolbar;
- Click that button: the transformations should reset and the button should disappear
